tests/activity_test.go

```diff
@@ -0,0 +1,37 @@
+// The MIT License
+//
+// Copyright (c) 2020 Temporal Technologies Inc.  All rights reserved.
```

Comment:
It's 2024.

Also wondering why this was required. Should we rename `activity.go` to `activity_test.go`?

Reply:
this is required because we have `suite` in activity.go. In order to run that suite we should add this file.